backbone/src/main/res/layout/rsb_image_capture_step_layout.xml

```diff
@@ -47,5 +47,14 @@
         app:layout_constraintTop_toBottomOf="@+id/imageCaptureStepQuestionItself"
         app:srcCompat="@drawable/rsb_ic_circle_8dp" />
 
+    <ImageButton
+        android:id="@+id/imageCaptureImageStepQuestionAnswerFullscreen"
+        style="@style/rsb_image_capture_fullscreen_style"
+        app:layout_constraintEnd_toEndOf="@+id/imageCaptureImageStepQuestionAnswer"
+        android:contentDescription="Full Screen"
+        android:background="@null"
+        app:layout_constraintTop_toTopOf="@+id/imageCaptureImageStepQuestionAnswer"
+        app:srcCompat="@drawable/rsb_ic_review_step_image_fullscreen" />
```

Comment:
You're using a PNG, any reason?

I would remove all this, and use the included in android (you have to import it from the Vector Graphics) "Crop Free" svg. It looks the same, it's a vector graphics, etc. 

Remove the imported PNGs.